---
description: What application can help me in MiGA @ XSEDE?
---

# Selection an Application

Different applications in [**MiGA @ XSEDE**](https://xsede.microbial-genomes.org/) automate commonly used workflows in MiGA. Explore the different tabs to identify the workflow that works for you.

{% hint style="info" %}
**Important:** Whenever possible, it's always better to submit a single task with many genomes instead of one task per genome
{% endhint %}

{% tabs %}
{% tab title="Indexing" %}
## Database Indexing

Index a custom MiGA database with a collection of genomes, including calculation of all-vs-all AAI and ANI matrixes.

{% page-ref page="applications/database-indexing.md" %}
{% endtab %}

{% tab title="Classification" %}
## Genome Classification

Genome-based typing of genomes within a well-characterized species.Classify genomes using the TypeMat MiGA database as a reference: All genomes from prokaryotic type material available in NCBI.

{% page-ref page="applications/genome-classification.md" %}
{% endtab %}

{% tab title="Typing " %}
## Genome Typing

Genome-based typing of genomes within a well-characterized species.

{% page-ref page="applications/genome-typing.md" %}
{% endtab %}

{% tab title="Dereplication" %}
## Genome Dereplication

Dereplicate a set of genomes using a threshold for ANI \(Average Nucleotide Identity\) or AAI \(Average Amino Acid Identity\).

{% page-ref page="applications/genome-dereplication.md" %}
{% endtab %}
{% endtabs %}



